src/Vixen.Common/AudioPlayer/PlayerFactory.cs

```diff
@@ -5,7 +5,15 @@ public class PlayerFactory
 		
 		public static IPlayer CreateNew(string fileName)
 		{
-			return new CoreAudioPlayer(fileName);
+			try
+			{
+				IPlayer player = new CoreAudioPlayer(fileName);
+				return player;
+			} catch
+			{
+				return null;
```

Comment:
Probably should at least log the exception and some message or just let the exception flow up to the higher level if it is trapped there.

Reply:
An error is loged on line 1819 of TimedSequenceEditorForm.cs. Do you also desire an additional logging line here?

Reply:
No need if it is trapped and logged higher up the stack.